### PR TITLE
zh-translation:content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md

### PR DESCRIPTION
**Pull request by vivian99-wu**
ref: https://github.com/servicemesher/istio-official-translation/issues/1632

### CODE REVIEW

File: content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md [hunk 1]
```diff
@@ -61,36 +60,34 @@ aliases:
...
-这是 _*.wikipedia.org_ 的情况。所有特定语言的网站都由 _wikipedia.org_ 服务器之一提供服务。您可以将流量路由到任意 _*.wikipedia.org_ 网站的 IP，包括 _www.wikipedia.org_，然后它将为任意特定网站[提供服务](https://en.wikipedia.org/wiki/Virtual_hosting)。
+能否配置通过 egress 网关访问 wildcard 主机取决于这组 wildcard 域名有唯一一个通用主机。
+以 _*.wikipedia.org_ 为例。每个语种特殊的站点都有自己的 _wikipedia.org_ 服务器。您可以向任意一个 _*.wikipedia.org_ 站点的 IP 发送请求，包括 _www.wikipedia.org_，该站点 [管理服务](https://en.wikipedia.org/wiki/Virtual_hosting) 所有特定主机。
 
-在一般情况下，单个主机服务器无法提供通配符的所有域名，
-需要更复杂的配置。
+通常情况下，通用域中的所有域名并不由一个唯一的 hosting server 提供服务。此时，需要一个更加复杂的配置。
 
-### 单个托管服务器的通配符配置{#wildcard-configuration-for-a-single-hosting-server}
+### Wildcard configuration for a single hosting server{#wildcard-configuration-for-a-single-hosting-server}
```
Comment (gauliang) [inline content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md]: 这里忘记翻译了

File: content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md [hunk 2]
```diff
@@ -204,32 +200,30 @@ $ kubectl delete virtualservice direct-wikipedia-through-egress-gateway
...
-在本节中，您部署的 egress gateway 在标准的 Istio Envoy 代理之外，还会部署一个 SNI 代理。此示例使用 [Nginx](http://nginx.org) 作为 SNI
-代理，但是，任何能够根据任意的、非提前配置的 SNI 值路由流量的 SNI 代理都可以使用。SNI 代理将会监听 `8443` 端口，您也可以使用任何端口，但需与指定给
-egress `Gateway` 的和 `VirtualServices` 绑定的端口不同。
-SNI 代理会将流量转发到 `443` 端口。
+本章节，除了标准的 Istio Envoy 代理，您将再部署一个带 SNI 代理的 egress 网关。本示例使用 [Nginx](http://nginx.org) 作为 SNI 代理。任何一个能够根据任意的、非预先配置的 SNI 值路由流量的 SNI 代理均可。
+SNI 代理将监听在端口 `8443` 上，您可以绑定任意其它端口，egress `Gateway` 和 `VirtualServices` 中配置的端口除外。SNI 代理将流量转发至端口 `443`。
 
-1.  为 Nginx SNI 代理创建一个配置文件。当需要时，您可能希望编辑该文件指定附加的 Nginx 配置。请注意，`server` 的 `listen` 指令指定端口 `8443`，
-   其 `proxy_pass` 指令使用 `ssl_preread_server_name` 和 `443` 端口及 `ssl_preread` 为 `on` 来启用 `SNI` 读取。
+1.  创建一个 Nginx SNI 代理的配置文件。您可以按需编辑文件指定附加的 Nginx 设置。注意 `server` 的 `listen` 元语指定端口为 `8443`，其 `proxy_pass` 元语使用 `ssl_preread_server_name` 端口为 `443`，`ssl_preread` 为 `on` 以开启 `SNI` 读。
```
Comment (gauliang) [inline content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md]: 这个地方，`元语`是专有名词还是笔误了？

File: content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md [hunk 3]
```diff
@@ -575,46 +571,46 @@ SNI 代理会将流量转发到 `443` 端口。
...
     {{< /text >}}
 
-    注意 `requestedServerName` 属性。
+    注意属性 `requestedServerName`。
 
-#### 清理任意域名的通配符配置{#cleanup-wildcard-configuration-for-arbitrary-domains}
+#### 清除任意域的 wildcard 配置{#cleanup-wildcard-configuration-for-arbitrary}
```
Comment (gauliang) [inline content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md]: #cleanup-wildcard-configuration-for-arbitrary-domains
参考 https://preliminary.istio.io/docs/tasks/traffic-management/egress/wildcard-egress-hosts/#cleanup-wildcard-configuration-for-arbitrary-domains

File: content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md [hunk 4]
```diff
@@ -1,31 +1,30 @@
...
 
-假设您希望在 Istio 中为 `wikipedia.org` 网站的所有语言版本启用 egress 流量。每个特定语言版本的 `wikipedia.org`
-都有自己的主机名，例如 `en.wikipedia.org` 和 `de.wikipedia.org` 分别对应英文和德文。
-您希望通过通用配置项对所有 `wikipedia` 网站启用 egress 流量，而无需单独配置每个语言的站点。
+假定您想要为 Istio 中所有语种的 `wikipedia.org` 站点开启 egress 流量。每个语种的 `wikipedia.org` 站点均有自己的主机名，譬如：英语和德语对应的主机分别为 `en.wikipedia.org` 和 `de.rikipedia.org`。您希望通过通用配置项开启所有 Wikipedia 站点的 egress 流量，无需单独配置每个语种的站点。
 
 {{< boilerplate before-you-begin-egress >}}
 
-*   [部署 Istio egress gateway](/zh/docs/tasks/traffic-management/egress/egress-gateway/#deploy-Istio-egress-gateway)。
+*   [部署 Istio egress 网关](/zh/docs/tasks/traffic-management/egress/egress-gateway/#deploy-istio-egress-gateway)。
```
Comment (tanjunchen) [inline content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md]: #deploy-istio-egress-gateway
->
#deploy-Istio-egress-gateway

Comment (vivian99-wu) [reply content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md]: 好的

File: content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md (hunk 3 above)
Comment (vivian99-wu) [reply content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md]: ok

File: content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md (hunk 2 above)
Comment (vivian99-wu) [reply content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md]: 笔误了，谢谢

File: content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md (hunk 1 above)
Comment (vivian99-wu) [reply content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md]: 好的

**Comment by tanjunchen** [comment]
@gauliang  can you merged ?

**Comment by gauliang** [comment]
> @gauliang can this pr be merged ?

/lgtm

**Pull request merged**